src/modules/75-cf/components/FlagDetailsOptionsMenuButton/__tests__/FlagDetailsOptionsMenuButton.test.tsx

```diff
@@ -321,9 +329,58 @@ describe('FlagDetailsOptionsMenuButton', () => {
     })
   })
 
-  describe('FlagOptionsButton - Edit', () => {
-    const isArchivingFfOn = false
+  describe('FlagDetailsOptionsMenuButton - Restore', () => {
+    const archivedMockFeature = cloneDeep(mockFeature)
+    archivedMockFeature.archived = true
+    test('it should render an RESTORE and DELETE button when FFM_7921_ARCHIVING_FEATURE_FLAGS is toggled ON and the flag is archived', async () => {
```

Comment:
Super minor, but try to make sure the beginning of the test doesn't get lost.
```suggestion

    test('it should render an RESTORE and DELETE button when FFM_7921_ARCHIVING_FEATURE_FLAGS is toggled ON and the flag is archived', async () => {
```